Source/carthage/Bootstrap.swift

```diff
@@ -26,7 +26,20 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				if let depsToUpdate = options.dependenciesToUpdate {
+					let resolvedCartfileContents = try! String(contentsOfURL: project.resolvedCartfileURL, encoding: NSUTF8StringEncoding)
+					let resolvedCartfile = ResolvedCartfile.fromString(resolvedCartfileContents).value!
+					
+					let resolvedDependencyNames = Set<String>(resolvedCartfile.dependencies.map { $0.project.name })
+					let updatedDependencyNames = Set<String>(depsToUpdate)
+					let unresolvedDependencyNames = updatedDependencyNames.subtract(resolvedDependencyNames)
```

Comment:
Oh, Swift you spoil us with your sweet standard library! Improved this according to your suggestion.